http-server-netty/src/main/java/io/micronaut/http/server/netty/binders/InputStreamBodyBinder.java

```diff
@@ -77,59 +93,61 @@ public InputStreamBodyBinder(HttpContentProcessorResolver processorResolver) {
                 PipedOutputStream outputStream = new PipedOutputStream();
                 try {
                     PipedInputStream inputStream = new PipedInputStream(outputStream);
-                    processor.subscribe(new CompletionAwareSubscriber<ByteBufHolder>() {
+                    Flux.from(processor)
+                        .onBackpressureBuffer()
```

Comment:
Wouldn't this lead to OOM?

Reply:
InputStream is not thread safe, so I don't see an alternative

Reply:
Then I don't think this is an improvement

Reply:
Provide a reproducer for your OOM theory in that case. "I don't think" is not a valid argument opposing a bug fix

Reply:
I will try to create the scenario. In the meantime, relax with your tone